stdlib/src/os/path/path.mojo

```diff
@@ -563,7 +564,7 @@ fn expandvars[PathLike: os.PathLike, //](path: PathLike) -> String:
         The expanded path.
     """
     var path_str = path.__fspath__()
-    var bytes = path_str.as_bytes().get_immutable()
+    var bytes = __type_of(path_str.as_bytes()).immut(path_str.as_bytes())
```

Comment:
**Question** Is there a way we might avoid the duplication here? I can see the potential value in having aliases for mutable/immutable variants of the current `Span`/`StringSlice` type, but in a case like this, a "normal" method call seems more readable at a glance.

Reply:
True, I removed the function mostly because I disliked the name. We could also change the method name to something like `.to_immut()` or directly `.immut()` (or leave that for another PR) WDYT?